cinema_service/urls.py

```diff
@@ -1,6 +1,7 @@
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
 
 urlpatterns = [
     path("admin/", admin.site.urls),
```

Comment:
add namespace and app_name to urls files

Reply:
good mark